src/Features/VisualBasic/Portable/UseAutoProperty/VisualBasicUseAutoPropertyCodeFixProvider.vb

```diff
@@ -63,19 +63,30 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.UseAutoProperty
                 statement = statement.WithAsClause(initializer.asNewClause)
             End If
 
+            If initializer.arrayBounds IsNot Nothing Then
+                Dim arrayCreation = SyntaxFactory.ArrayCreationExpression(
+                    Nothing,
+                    DirectCast(statement.GetReturnType(), ArrayTypeSyntax).ElementType,
```

Comment:
how do you that it's safe to do `DirectCast(statement.GetReturnType(), ArrayTypeSyntax)`?

Reply:
would far prefer you use semantics to get this type, and then do `.GenerateTypeSyntax()` from that.